mylyn.tasks/connectors/bugzilla-rest/org.eclipse.mylyn.bugzilla.rest.core.tests/src/org/eclipse/mylyn/bugzilla/rest/core/tests/BugzillaRestClientTest.java

```diff
@@ -128,12 +128,14 @@ public AbstractTestFixture getActualFixture() {
 	}
 
 	@Test
+	@ConditionalIgnoreRule.ConditionalIgnore(condition = MustRunOnCIServerRule.class)
```

Comment:
I would prefer going straight to JUnit5 end @EnabledIf instead, like I did with #589

Reply:
Yes, we can do that. BugzillaRest Connector is special because we use here a Junit4TestFixtureRunner with a BlockJUnit4ClassRunner to group the unit test per fixture.

Have you the migration of the BugzillaRest Connector on your list or should I do this because I implement the Connector?